Some Details

luboskris opened the pull request.

Commit:
Details

Commit:
Merge branch 'master' of https://github.com/luboskris/railway

vsvvan commented on src/components/Order.tsx:
```diff
@@ -71,15 +73,9 @@ export const Order = ({ trainInfo, userData }: Props) => {
-                    )
-                  </Typography>
-                  <Typography>{userData.passengerInformation.email}</Typography>
+                  <Typography>Name Surname</Typography>
```
Why did you change this part??

luboskris replied:
?? What we need to put there??

vsvvan replied:
Just remove changes that you did in this file. All changes that you did here are not correct we should use information  from userData object as it was before
You can see what are the difference if you'll go to the order overview page, with your changes there will be Name Surname at user info tab instead of correct name from user.json file (Jakub Liptak or smt like this)

vsvvan commented on src/components/Order.tsx:
```diff
@@ -160,7 +150,7 @@ export const Order = ({ trainInfo, userData }: Props) => {
+                Payment
               </Button>
             </div>
           </div>
```
Questions refers to all changes that was made in this file, you revert last changes that were actually correct. Fix this please

vsvvan replied:
Rollback all changes that were made in this file

vsvvan commented on src/components/App.css:
```diff
@@ -162,6 +162,13 @@ footer li::before {
   margin-left: 7%;
   display: flex;
 }
+.Authors_info {
```
fix name case, take a look at example
.DayStyle {
##
}

luboskris replied:
??  same as with Menu-bar?

Commit:
l

Commit:
Merge pull request #3 from vsvvan/master

fix page title

Commit:
k

Commit:
Merge branch 'master' of https://github.com/luboskris/railway

vsvvan commented on src/components/App.css:
```diff
@@ -11,6 +11,79 @@
+  text-align: center;
+  background: #0773d7;
+}
+.Menu-bar{
```
should be .MenuBar, the same for others, recheck please

vsvvan commented on src/components/App.css:
```diff
@@ -11,6 +11,79 @@
+  margin: 10px;
+  }
+
+.Sub-menu-1
```
SubMenu1

vsvvan commented on src/components/HeaderMainPage.tsx:
```diff
@@ -0,0 +1,24 @@
+import { useNavigate } from 'react-router-dom';
+import { Button } from '@mui/material';
+
+export const Header = () => {
```
rename const to the name of the file -> HeaderMainPage and don't forget to rename in place where you use it

vsvvan commented on src/components/App.css:
```diff
@@ -11,6 +11,79 @@
+.Sub-menu-1 li::before a{
+  display: inline-block;
+  }
+.menu-list{
```
.MenuList

vsvvan commented on src/components/Header.tsx:
```diff
@@ -8,17 +8,50 @@ export const Header = () => {
+          <ul>
+          <li><a href="#" className='click'>Ticket Purchase</a>
+            <div className='Sub-menu-1'>
+            <ul className='menu-list'>
```
reuse menu component from mui library, examples of use are here 
https://mui.com/material-ui/react-menu/#basic-menu

Commit:
k

Commit:
k

Related to menu. Sincirely, i don´t know how to do it, because it is out of my depth (React) and I tried several solutions but they didn´t work. So, i leave only clickable menu. and related to delete things.

q401367 committed:
atomic commit

q401367 committed:
atomic commit
